librz/core/cmd/cmd_open.c

```diff
@@ -1519,3 +1455,128 @@ RZ_IPI RzCmdStatus rz_open_prioritize_next_rotate_handler(RzCore *core, int argc
 	}
 	return prioritize_file(core, fd) ? RZ_CMD_STATUS_OK : RZ_CMD_STATUS_ERROR;
 }
+
+RZ_IPI RzCmdStatus rz_open_maps_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	return RZ_CMD_STATUS_ERROR;
+}
+
+RZ_IPI RzCmdStatus rz_open_maps_remove_handler(RzCore *core, int argc, const char **argv) {
+	if (!rz_num_is_valid_input(NULL, argv[1])) {
+		RZ_LOG_ERROR("Invalid map id '%s'\n", argv[1]);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	ut32 map_id = rz_num_math(NULL, argv[1]);
+	if (!rz_io_map_del(core->io, map_id)) {
+		RZ_LOG_ERROR("Could not delete IO map %d\n", map_id);
+		return RZ_CMD_STATUS_ERROR;
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_open_maps_remove_all_handler(RzCore *core, int argc, const char **argv) {
+	rz_io_map_reset(core->io);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_open_maps_current_handler(RzCore *core, int argc, const char **argv) {
+	return RZ_CMD_STATUS_ERROR;
```

Comment:
I guess it's not finished yet, right?

Reply:
I should remove this. I can't convert the `om` command itself unless i convert all of the `om` subcommands first.